nixos/modules/config/users-groups.nix

```diff
@@ -960,11 +966,14 @@ in {
           (filter (x: x != null) (map (flip getAttr user) passwordOptions));
       in optional (!unambiguousPasswordConfiguration) ''
         The user '${user.name}' has multiple of the options
-        `hashedPassword`, `password`, `hashedPasswordFile`, `initialPassword`
-        & `initialHashedPassword` set to a non-null value.
+        `initialHashedPassword`, `hashedPassword`, `initialPassword`, `password`,
+        & `hashedPasswordFile` set to a non-null value.
         The options silently discard others by the order of precedence
         given above which can lead to surprising results. To resolve this warning,
         set at most one of the options above to a non-`null` value.
+        NOTE: If `systemd.sysusers.enable = true;` then the the options and order
+        of precedence are: `initialPassword`,  `initialHashedPassword`,
+        & `hashedPasswordFile`.
```

Comment:
What about the other two options?

Reply:
The other two options aren't supported when using `systemd.sysusers.enable`. I changed that to use some other (hopefully more clear) wording from elsewhere. So should be addressed, but let me know if you still think it's too unclear.